dir.targets

```diff
@@ -166,14 +166,22 @@
 
   <Target Name="Test"
           DependsOnTargets="Build;EnsurePrerequisitesCopied"
-          Condition="'$(IsTestProject)' == 'true'">
+          Condition="'$(IsTestProject)' == 'true' and '$(RunTests)' != 'false'">
     <!-- Exec the console runner instead of using the MSBuild runner,
          because our tests have dependencies on being run as x86. -->
     <!-- Noshadow because our assemblies do not pass the checks for it
          (due to open source signing). -->
     <!-- No parallel execution because most of our tests do bad things
          with state.  They were passing because MSTest was serial. -->
-    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt" />
+
+    <!-- Run tests on the full framework -->
+    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt"
+          Condition="'$(NetCoreBuild)' != 'true'" />
```

Comment:
Currently we're not setting `NetCoreBuild` in the false case, so it'll get evaluated to an empty string instead of `false`.  For bools in MSBuild I tend to prefer testing only one side, and that's what we're doing for this (all existing checks are `== 'true'` or `!= 'true'`).

In principle I prefer the explicit model (I'd go for an equivalent of Perl's `use strict;`) but we'd need a way to ensure that a property got set on all codepaths which is ugly.